build_tools/bump_submodules.py

```diff
@@ -170,7 +235,32 @@ def run(args: argparse.Namespace, fetch_args: list[str], system_projects: list[s
             ["git", "push", "-u", "origin", args.branch_name],
             cwd=THEROCK_DIR,
         )
```

Comment:
Let's add an additional option for `--create-pr`  instead of having `--push-branch` always create a PR